src/arch/xtensa/include/arch/alloc.h

```diff
@@ -6,14 +6,17 @@
  */
 
 /**
- * \file arch/xtensa/smp/include/arch/alloc.h
- * \brief Xtensa SMP memory allocation header file
+ * \file arch/xtensa/include/arch/alloc.h
+ * \brief Xtensa memory allocation header file
  * \authors Tomasz Lauda <tomasz.lauda@linux.intel.com>
  */
 
 #ifndef __ARCH_ALLOC_H__
 #define __ARCH_ALLOC_H__
 
+#include <config.h>
+
+#if CONFIG_SMP
```

Comment:
Isn't `CONFIG_SMP` removed at all? I see it being removed from `src/arch/xtensa/Kconfig`, but maybe I missed that it's moved somewhere else.

Reply:
Nvm, I see it's just now helper config.

Reply:
It wasn't removed in the first patch but moved below the CORE_COUNT config, and its default value is now saner (defaults to SMP build when CORE_COUNT > 1).

Reply:
@paulstelian97 Yea, i noticed it later ;p